config/prod.exs

```diff
@@ -0,0 +1,12 @@
+use Mix.Config
+
+port =
+  case System.get_env("PORT") do
+    port when is_binary(port) -> String.to_integer(port)
+    nil -> 80 # default port
+  end
```

Comment:
I was doing it like that before, I changed here: https://github.com/samphilipd/hook_line_and_slacker/commit/d893b8c2ebebe89b9ac5fa3046da48bc37633827

I'm not particularly attached to either method, it just seemed neater like this.